test/cmd/newapp.sh

```diff
@@ -12,6 +12,7 @@ os::log::install_errexit
 # This test validates the new-app command
 
 os::cmd::expect_success_and_text 'oc new-app library/php mysql -o yaml' '3306'
+os::cmd::expect_success_and_text 'oc new-app library/php mysql --dry-run' "Image \"mysql\" runs as the 'root' user which may not be permitted by your cluster administrator"
```

Comment:
really minor nit... this test is not consistent with the test below... either test for the full string on both or just test for the last part "runs as the 'root' user..." in both.